src/moveroplot/config/plot_settings.py

```diff
@@ -0,0 +1,18 @@
+"""Static configurations settings for plots."""
+
+
+class PlotSettings:
```

Comment:
I understand that the class acts as a container for the global variables that are implemented as class variables but the same could be achieved with module variables. Since the class does not have any other purpose, I would say that it's not necessary to have it. The module can also be used to namespace the variables to the maintain the organisation.